### DISCUSSION TITLE
MSSQL - updated_at [Jun 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2472] [Discord Threads]
> 🙋‍♂️ **onlyspike**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### it should be update - was the table created using noco GUI ? We create a trigger when table is created  
#  
> 🙋‍♂️ **onlyspike**  
#### Thank you for your response. I can't remember to be honest. However, I just created a new table via the GUI, added a new row via the GUI and then did a patch on swagger and it still didn't update.  
#  
> 🙋‍♂️ **onlyspike**  
#### Is there any feedback regarding this?  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb/issues/2379  
#  
> 💁 **o1lab**  
#### have raised an issue for you  
#  
> 💁 **o1lab**  
#### pls mention what version of MSSQL are you using  
#  
> 🙋‍♂️ **onlyspike**  
#### Thank you. SQL Server 2016 Standard, version 14.0.2037.2  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/985907280652406855)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
